NEWS.md

```diff
@@ -1,4 +1,6 @@
 # riskassessment (development version)
+* Fixed busted button introduced with #547 (#592)
```

Comment:
Hi @Robert-Krajcik, this bullet point shouldn't be moved, correct? It belongs under "Squashed Bugs" for the `v2.0.0` release.

Reply:
OK, moved it back